### PR TITLE
Docs 3236 document firewall rules config vh

### PR DESCRIPTION
**Pull request by valkyrie69**
Added a little bit more context and steps for configuring firewall rules in the UI.

### CODE REVIEW

**Review by henrymollman — APPROVED** [review]

File: modules/ROOT/pages/virtual-private-cloud.adoc [hunk 1]
```diff
@@ -5,10 +5,10 @@ endif::[]
 
 The Virtual Private Cloud (VPC) offering allows you to create a virtual, private, and isolated network segment in the cloud to host your CloudHub workers.
 
-By connecting to your CloudHub VPC you can extend your corporate intranet whether on-prem data centers through a secured VPN tunnel, or one of your private AWS VPCs through a VPN peering, or using AWS direct connect. This allows CloudHub workers to access resources behind your corporate firewall. +
+By connecting to your CloudHub VPC you can extend your corporate intranet whether on-premises data centers through a secured VPN tunnel, or one of your private AWS VPCs through a VPN peering, or using AWS direct connect. This allows CloudHub workers to access resources behind your corporate firewall. +
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/virtual-private-cloud.adoc]: This sentence is a bit long/hard to parse. Maybe rearrange the two sentences so something like "Connecting..your corporate intranet allows CloudHub workers to access.... You can connect whether you use on-premise data centers through a tunnel..." (separate the two key ideas: you can do this, and you can do this in three different ways"

File: modules/ROOT/pages/virtual-private-cloud.adoc [hunk 2]
```diff
@@ -5,10 +5,10 @@ endif::[]
...
 Keep in mind that you can't reuse your existing AWS VPC as a CloudHub VPC.
 
-The base VPC subscription includes 2 VPCs and one VPC can be associated with multiple environments. This allows you, for example, to have one isolated network for your production environment, and another for your QA and staging.
+The base VPC subscription includes two VPCs and one VPC can be associated with multiple environments. This allows you, for example, to have one isolated network for your production environment, and another for your QA and staging.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/virtual-private-cloud.adoc]: to get rid of the "garden path" I'd change "and" to something like "...two VPCs. One of the VPCs can be associated..."

File: modules/ROOT/pages/vpc-firewall-rules-concept.adoc [hunk 3]
```diff
@@ -31,9 +31,9 @@ When creating a VPC, 4 firewall rules are created by default:
 },
 ----
 +
-These firewall rules allow traffic from the VPC to reach your workers through ports 8091 and 8092. These are the only ports used by your CloudHub dedicated load balancer to proxy all external communications to your workers through.
+These firewall rules allow traffic from within the VPC to reach your workers through ports 8091 and 8092. These are the only ports used by your CloudHub dedicated load balancer to proxy all external communications to your workers.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/vpc-firewall-rules-concept.adoc]: since CloudHub dedicated modifies balancer, should it be hyphenated? CloudHub-dedicated load balancer?

File: modules/ROOT/pages/vpc-firewall-rules-concept.adoc [hunk 4]
```diff
@@ -57,9 +57,48 @@ These ports are used by CloudHub's shared load balancer to proxy external reques
...
+.. Click *Add New Rule*.
+.. Select the following values for the type and source in the corresponding columns: +
+* *Type*: Select the transport protocol type for the rule.
+* *Source*: Select the source IP address, or select *Custom* to define your own source.
+.. Enter a value for *Port Range* and click *Add to list*.
+. To delete a firewall rule, hover over the firewall rule row to delete, then click the trashcan icon in the far-right column. +
+
+[NOTE]
+--
+There is not a confirmation dialog when you delete a firewall rule, so make sure you really want to delete it.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/vpc-firewall-rules-concept.adoc]: Active voice here? Something like No confirmation dialog displays (arguably more active?)"

File: modules/ROOT/pages/vpc-provisioning-concept.adoc [hunk 5]
```diff
@@ -5,62 +5,66 @@ endif::[]
...
-. Connecting to your VPC. +
-This needs to be done by a MuleSoft representative.
+. Connect to your VPC. +
+This must be done by a MuleSoft representative.
 
-See the tutorials and task in the See Also section below for detailed information about these processes. +
-However, before creating the VPC, you need to be aware that some settings are only configurable during the first set up:
+[NOTE]
+--
+To avoid unnecessary downtime, it is critical to define the VPC settings during the initial setup of the VPC. +
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/vpc-provisioning-concept.adoc]: does "initial setup" = "Create the VPC" step? Any way to make this clear (by mentioning "setup" or "configure" in the description of that step?

File: modules/ROOT/pages/vpc-provisioning-concept.adoc [hunk 6]
```diff
@@ -5,62 +5,66 @@ endif::[]
...
+* For better fault tolerance, the VPC subnet may be divided into up to four Availability Zones.
 * A few IP addresses are reserved for infrastructure.
 * At least two IP addresses per worker to perform at zero-downtime.
 
 Due to this structure, the smallest network subnet block you can assign for your VPC is /24 and the largest /16. +
 A /24 CIDR notation subnet has 256 IP addresses. +
-Reserving one IP address for the network and one for broadcast you have 254 hosts for your worker, which get divided into 4 availability zones of 62 hosts each. Consider that each worker potentially needs two IP addresses for zero-downtime, that's around 30 IP addresses to assign to your workers. +
-Consider also the environments that would require in a VPC.  For example, f you need space for 300 instances each in Dev, QA and UAT you need to size for 1024 instances at minimum.
+Reserving one IP address for the network and one for broadcast leaves you with 254 hosts for your worker. These get divided into four availability zones of 62 hosts each. Consider that each worker potentially needs two IP addresses for zero-downtime, which leaves around 30 IP addresses to assign to your workers. +
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/vpc-provisioning-concept.adoc]: I don't think zero downtime needs a hyphen at the end of the clause?

File: modules/ROOT/pages/vpc-tutorial.adoc [hunk 7]
```diff
@@ -25,7 +25,7 @@ In this case, the VPC is bound to `US East (Virginia)`.
 +
 [NOTE]
 --
-All VPCs need to be associated to a CloudHub region.
+All VPCs must be associated to a CloudHub region.
```
Comment (Mulesoft-Mysti) [inline modules/ROOT/pages/vpc-tutorial.adoc:28]: associated with instead of to?

**Review by Mulesoft-Mysti — APPROVED** [review]
I made some optional suggestions. Looks great!

File: modules/ROOT/pages/vpc-tutorial.adoc (hunk 7 above)
Comment (valkyrie69) [reply modules/ROOT/pages/vpc-tutorial.adoc:28]: Yes, but I think I fixed this in a different branch.

File: modules/ROOT/pages/vpc-provisioning-concept.adoc (hunk 5 above)
Comment (valkyrie69) [reply modules/ROOT/pages/vpc-provisioning-concept.adoc]: I put ...when you first create a new VPC.
Create VPC is the terminology used in the UI.
